src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleTestBase.cs

```diff
@@ -47,6 +47,7 @@ public static string UseFrameworkDependentHost(TestProjectFixture testFixture)
             Version targetFrameworkVersion = null)
         {
             UseSingleFileSelfContainedHost(testFixture);
+            options |= BundleOptions.EnableCompression;
```

Comment:
We should also add a couple of tests which disable compression on purpose. I know we expect compression to be on by default, but will allow users to disable it, so we should validate that it works.

Reply:
Some files are not compressed, and .json never will be, so the case with uncompressed files is validated. 
Turning off compression in one of self-contained tests is an easy thing to do though.